app/pages/cif-external/index.tsx

```diff
@@ -0,0 +1,116 @@
+/* eslint-disable relay/graphql-naming */
+import ExternalLayout from "components/Layout/ExternalLayout";
+import ProjectTableRow from "components/Project/ProjectTableRow";
+import Table from "components/Table";
+import {
+  NoHeaderFilter,
+  SearchableDropdownFilter,
+  TextFilter,
+} from "components/Table/Filters";
+import withRelayOptions from "lib/relay/withRelayOptions";
+import { useMemo } from "react";
+import { graphql, usePreloadedQuery } from "react-relay/hooks";
+import { RelayProps, withRelay } from "relay-nextjs";
+import { cifExternalQuery } from "__generated__/cifExternalQuery.graphql";
+
+export const ExternalProjectsQuery = graphql`
+  query cifExternalQuery(
+    $projectName: String
+    $proposalReference: String
+    $status: String
+    $offset: Int
+    $pageSize: Int
+    $orderBy: [ProjectsOrderBy!]
+  ) {
+    session {
+      ...ExternalLayout_session
+    }
+    allProjects(
+      first: $pageSize
+      offset: $offset
+      filter: {
+        projectName: { includesInsensitive: $projectName }
+        proposalReference: { includesInsensitive: $proposalReference }
+        projectStatusByProjectStatusId: {
+          name: { includesInsensitive: $status }
+        }
+        # hardcoding an operator so that everyone who tests and demos this can have the same experience
```

Comment:
This was suggested in https://app.zenhub.com/workspaces/climate-action-secretariat-60ca4121764d710011481ca2/issues/gh/bcgov/cas-cif/1411